packages/bpk-component-content-cards/src/ContentCard/ContentCard.module.scss

```diff
@@ -0,0 +1,92 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable selector-max-compound-selectors */
+@import '~bpk-mixins/index';
+
+$bpk-spacing-v2: true;
+
+@mixin aspect-ratio($width, $height) {
+  position: relative;
+
+  &::before {
+    content: '';
+    display: block;
+    width: 100%;
+    padding-top: ($height / $width) * 100%;
+  }
+
+  > :first-child {
+    position: absolute;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    left: 0;
+  }
+}
+
+.bpk-content-cards {
+  &__card-link {
+    color: inherit;
+    font: inherit;
+    text-decoration: inherit;
+  }
+
+  &__card-horizontal {
+    display: grid;
+    column-gap: bpk-spacing-xxl();
+    grid-template-columns: 50% 40%;
+    align-items: center;
+
+    @include bpk-breakpoint-mobile {
+      grid-template-columns: 1fr;
+    }
+  }
+
+  &__card-horizontal &__image-aspect-ratio {
+    width: 100%;
+    height: 100%;
+
+    @include bpk-breakpoint-mobile {
+      max-height: $bpk-one-pixel-rem * 180;
```

Comment:
Is there a way we can use other multiple of spacing tokens rather than just of 1 * x?

Reply:
Just wondering for future work as well, how is it better than having absolute values?
My thinking being that in this case, with design handover we see the `px` in Figma and can easily work it out here.
With spacing tokens (not clear which ones to use, `xs`, `sm`, etc) it gets complicated. If spacing gets updated, I don't think we want such values to change to some new number – it can cause more evil than bad.

Reply:
> If spacing gets updated, I don't think we want such values to change to some new number – it can cause more evil than bad.

In this case we would be making the changes after they are passed from design so if spacing values change we would have action to update all our components as part of that

So I don't think there would be any bad here - it just makes it looks clearer rather than what we call 'magic' numbers instead. If spacing changes anyway this value wouldn't be valid anyway as the system would change so doing it as suggested would be the preferred way of doing it.

>My thinking being that in this case, with design handover we see the px in Figma and can easily work it out here.

If this was the case then it would basically say there is no point in having other spacing values and not even using rem, and just hardcoding the px values. Design should be guiding in this or it will be easy to work out when used in conjunction with the following page: https://backpack.github.io/tokens/spacing#web

Reply:
something like `$bpk-spacing-base-v-2 * 11.25` to make it easier to read it as rem values?

Reply:
or `$bpk-spacing-md * 10` ?

Reply:
>in having other spacing values and not even using rem, and just hardcoding the px values

It does make sense for margins and paddings, font sizes, etc, where consistent values makes sense. 
But for the cases like this I really believe we should allow just `px` values. And we (all devs) do `$bpk-one-pixel-rem * 180` only to avoid `stylelint` complains. 
`$bpk-spacing-base-v-2 * 11.25` or any spacing value doesn't make much sense to anyone who looks at the code. This looks much more magical than just `px` value, because `px` value can be implied from Figma file.

Reply:
Ok so should I keep it at `$bpk-one-pixel-rem * 180` then?